apps/recovery-relay/components/WithdrawModal/index.tsx

```diff
@@ -127,7 +127,9 @@ export const WithdrawModal = () => {
                         .get(inboundRelayParams?.accountId)
                         ?.wallets.get(inboundRelayParams?.signedTx.assetId);
 
-                      const derivation = wallet?.derivations?.get(inboundRelayParams?.signedTx.from);
+                      const derivation = wallet?.derivations?.get(
+                        `${inboundRelayParams?.signedTx.assetId}-${inboundRelayParams?.signedTx.from}`,
```

Comment:
Move this to a separate function in the shared-package and all instances where we do X-Y format should call that function with the parameters instead. This way if we need to alter this in the future we can without having to go through each location it is used.

Reply:
And remove comment